daemon/service/service_nocgo.go

```diff
@@ -0,0 +1,9 @@
+// +build !cgo
+
+// Package service implements the daemon service for mediating access to the
+// storage backend.
+package service
+
+// A dummy implementation of Serve for the environment of Cgo being disabled.
+func Serve(sockpath, dbpath string) {
```

Comment:
Ah. I was suggesting that you put `select {}` in your dummy Serve implementation just for the sake of making it behave more like the real Serve, and add a logging for easier debugging.